R/extra.R

```diff
@@ -2,13 +2,20 @@
 #' getting the same observation twice. Need to improve this help file.
 #'
 #' @inheritParams global_arguments
+#'
+#' @param nTrain Number of training observations to sample from.
+#'
+#' @param nTest Number of test observations to sample from.
+#'
+#' @param nosamp Number of samples
+#'
```

Comment:
Could you add which type each of the param's is? 

I.e. if nTrain is should be a positive integer vector, do 
```
#' @param nTrain Positive integer. Number of training observations to sample from.
```